pkg/services/publicdashboards/service/service.go

```diff
@@ -142,11 +142,6 @@ func (pd *PublicDashboardServiceImpl) Create(ctx context.Context, u *user.Signed
 		return nil, err
 	}
 
-	// set default value for time settings
-	if dto.PublicDashboard.TimeSettings == nil {
```

Comment:
why'd this get moved down below?

Reply:
Mostly to keep together related code and because I prefer to validate the input before mutating it with default values.
Validations go first and sometimes this initialization code would not get executed.